### PR TITLE
core-svc-feat(master-data-service): add archives list mds endpoint

### PR DESCRIPTION
**Pull request by rachadiannovansyah**
### Overview
- add API endpoint list archives with method `GET`

cc: @jabardigitalservice/jds-backend 

Evidence
project: Portal Jabar
title: add archives list master data services endpoint
participants: @rachadiannovansyah @rindibudiaramdhan @ayocodingit @iqbal167 @indraprasetya154

### CODE REVIEW

**Review by rindibudiaramdhan — APPROVED** [review]
no paginating? 😋

File: core-service/src/modules/master-data-service/usecase/mds_ucase.go [hunk 1]
```diff
@@ -176,3 +176,17 @@ func (n *masterDataServiceUsecase) TabStatus(ctx context.Context) (res []domain.
 	}
 	return
 }
+
+func (n *masterDataServiceUsecase) Archive(c context.Context, params *domain.Request) (
+	res []domain.MasterDataService, err error) {
+	ctx, cancel := context.WithTimeout(c, n.contextTimeout)
+	defer cancel()
+
+	params.Filters["status"] = "ARCHIVE"
```
Comment (ayocodingit) [inline core-service/src/modules/master-data-service/usecase/mds_ucase.go:185]: better, are you set constant for `ARCHIVE`?

**Comment by rachadiannovansyah** [comment]
> no paginating?

yes, this list is dedicated for drop-down list selection on publication-services. Has been discussion for our FE teams.

File: core-service/src/modules/master-data-service/usecase/mds_ucase.go (hunk 1 above)
Comment (rachadiannovansyah) [reply core-service/src/modules/master-data-service/usecase/mds_ucase.go]: even though only used in one func but it's good to make it constant value.
here we go https://github.com/jabardigitalservice/portal-jabar-services/pull/423/commits/fa0bf06fc9a940a7849bbc5ffdbc276dfa42ef53

**Review by iqbal167 — APPROVED** [review]

**Review by ayocodingit — APPROVED** [review]

**Review by indraprasetya154 — APPROVED** [review]

**Pull request merged**